gpkit/user_scripts.py

```diff
@@ -5,49 +5,6 @@
 from .nomials import Posynomial
 from .variables import Variable, VectorVariable
 from .posyarray import PosyArray
-from .geometric_program import GeometricProgram
-
-
-def make_feasibility_gp(gp, varname=None, flavour="max"):
```

Comment:
This moved to being a method of GP. I think we might want a similar method for Model at some point.